jsx/PaginationLinks.js

```diff
@@ -1,148 +1,135 @@
-/* exported RPaginationLinks */
+import React, {useState} from 'react';
 
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
+const theme = {
+  primary: '#053665',
+  secondary: '#A6D3F5',
+  accent: '#E89A0C',
+  neutral: '#DDD',
+};
 
 /**
- * Pagination component
+ * Produces the pagination number links
+ *
+ * @param {jsx} children
+ * @param {bool} active
+ * @param {func} onClick
+ *
+ * @return {jsx}
  */
-class PaginationLinks extends Component {
-  /**
-   * @constructor
-   * @param {object} props - React Component properties
-   */
-  constructor(props) {
-    super(props);
-
-    this.state = {
+function PaginationLink({
+  children,
+  active,
+  onClick,
+}) {
+  const [hover, setHover] = useState(false);
+  const hoverOn = () => setHover(true);
+  const hoverOff = () => setHover(false);
+  const style = {
```

Comment:
@HenriRabalais I haven't tested the following suggestion on Loris but we might benefit from moving towards css classes. The top of the file can have an import of the css file needed and then we could use classNames instead of passing style objects. Ex:
`import '../../htdocs/css/Pagination.css';`

Reply:
I forgot that you're a vim user. I don't want to propose anything where the navigation needed will slow anyone down. It's somewhat fast using phpstorm to jump to where the css is located. I've been using the css classes in another project associated with Loris and it has been much nicer by having cleaner code (removing styling from the code logic). That's why I thought about it when I saw your PR. Maybe it's best to leave the idea out of this PR and just bring the discussion to a Loris meeting?

Reply:
Yeah, I think the principles I outlined still hold for me even if I wasn't using vim, but I could be convinced otherwise since there really is no consensus on this and it does boil down to our choice as a team.

I would be happy to discuss it at a loris meeting. Here's an article that I believe outlines our three options:
1. Vanilla CSS (what we currently use)
2. CSS Module (what you're proposing)
3. CSS in JS ( what I'm proposing)

https://jsramblings.com/understand-the-react-styling-paradigms/

Reply:
@HenriRabalais I'm okay with continuing to use inline styling. One article for possibilities of not using inline styling 'always' would be: https://www.sderosiaux.com/articles/2015/08/17/react-inline-styles-vs-css-stupid-benchmark/

Reply:
Hmm...  yeah, this is definitely concerning when it comes to the datatable.

<img width="1150" alt="Screen Shot 2021-03-30 at 00 23 32" src="https://user-images.githubusercontent.com/7550451/112907483-ededc980-90bb-11eb-866b-f189a373f5e4.png">

Reply:
@HenriRabalais also the memory impact for users with not much ram.
```
Inline styles take way more size in the DOM, are converted more slowly from VDOM (have probably a bigger impact on memory), and take more time to be handled by the browser.

But they have no impact on performance once it’s rendered.
```